js/react_native/e2e/android/app/src/main/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
+    <domain-config cleartextTrafficPermitted="true">
+        <domain includeSubdomains="true">10.0.2.2</domain>
```

Comment:
what is this for?

Reply:
explained in this comment:) https://github.com/microsoft/onnxruntime/pull/15133#discussion_r1157853665